hs_discover/views.py

```diff
@@ -260,12 +280,8 @@ def get(self, request, *args, **kwargs):
                 "geo": pt
             })
 
-        if sort == 'title':
-            resources = sorted(resources, key=lambda idx: idx['title'].lower())
-        elif sort == '-title':
-            resources = sorted(resources, key=lambda idx: idx['title'].lower(), reverse=True)
-
         return JsonResponse({
```

Comment:
this was in the wrong place and was sorting within the page and not across all thousands of resources. The index is missing 'title' exact and 'title' alone gives undesirable results (NLP rooting). Alva will submit a fix outside of this release. For now slow python sorting is used.